### ISSUE TITLE
Inbox keeps getting marked as unread

### ISSUE THREAD

**Issue by kghorvath** [Livinglist/Hacki#361]
Every few days or so the inbox tab keeps showing all my replies as new and unread, even though I keep re-marking them as read. I'm not quite sure what is triggering this.

**Comment by Livinglist** [comment]
Hi @kghorvath which version of Hacki are you using? Was this on iOS or Android?

**Comment by kghorvath** [comment]
This is 2.6.1 on iOS 17.2.1. The logs don't have much:

`[I]  updating tabs to [StoryType.top, StoryType.best, StoryType.latest, StoryType.ask, StoryType.show]
[I]  Resetting comments in CommentCache
[I]  Resetting comments in CommentCache
[I]  Resetting comments in CommentCache
Current session logs:
[I]  updating tabs to [StoryType.top, StoryType.best, StoryType.latest, StoryType.ask, StoryType.show]`

**Comment by kghorvath** [comment]
Just happened again today since I opened the issue. This is what it looks like:![image](https://github.com/Livinglist/Hacki/assets/38924930/250fd113-8f45-40f8-a7d8-ae250e3311b4)

**Comment by Livinglist** [comment]
I have been looking into this issue but still have no luck with it. Very hard to reproduce, in the mean time, pls try turning off the notification on new reply inside Hacki’s app settings.

**Comment by kghorvath** [comment]
No worries, I'm not exactly sure how to reproduce it either.

**Comment by Markus40** [comment]
I have the same problem.
I use Android 14 and I am pretty sure it occured in 2.8.1.

**Comment by mitanshu7** [comment]
When I reinstalled hacki and logged on, all the replies were also marked as unread. 

Could this point to the issue?

**Comment by Markus40** [comment]
> Hi [@kghorvath](https://github.com/kghorvath) [@Markus40](https://github.com/Markus40) [@mitanshu7](https://github.com/mitanshu7) , is this still an issue in recent releases?

Yes, it happened recently, but I don't know the version number. I'll let you know next time if I remember.

**Comment by Markus40** [comment]
> Hi [@kghorvath](https://github.com/kghorvath) [@Markus40](https://github.com/Markus40) [@mitanshu7](https://github.com/mitanshu7) , is this still an issue in recent releases?

It just happened again with v3.6.4

**Comment by Markus40** [comment]
@Livinglist I can reproduce the problem by clearing the cache and restarting the app.
